tools/libxl/libxl_internal.c

```diff
@@ -575,6 +575,24 @@ void libxl__update_domain_configuration(libxl__gc *gc,
     dst->b_info.video_memkb = src->b_info.video_memkb;
 }
 
+int libxl__is_driver_domain(libxl__gc *gc, uint32_t domid)
```

Comment:
Sounds like returning bool is more cleaner, no?

Reply:
Sure. Done.